cluster/examples/kubernetes/ceph/storageclass-ec.yaml

```diff
@@ -1,3 +1,9 @@
+#################################################################################################################
+# Create a storage class with a data pool that uses erasure coding for a production environment.
+# A metadata pool is created with replication enabled. A minimum of 3 OSDs are required in this example.
```

Comment:
I think it should be noted that the OSDs need to according to the `failureDomain` (default `host`).
One of the other yaml files mentions that but just uses the default for the `failureDomain: host`.

Reply:
Sure, I'll mention the failureDomain